### DISCUSSION TITLE
Is it necessary to restart the api after indexing new data

### DISCUSSION THREAD

**Discussion by MALKARAJ** [mediagis/nominatim-docker#371] [General]
Hi Team,
Thanks for doing the great work keep it up,

I have an issue, is it necessary to stop and start the API after every new indexed data ,
kindly give me details about this

**Comment by mtmail** [comment]
(Please use the discussions instead for questions https://github.com/mediagis/nominatim-docker/discussions)

The API doesn't have to be restarted when new data is indexed. The indexing runs in the background and users can do searches during the indexing. Some people do indexing every minute. Of course if it's the initial import then the database is mostly empty (or unindexed) and the API will return no results.

**Comment by MALKARAJ** [comment]
Hi thanks for your response,
i have an another doubt ,
By updating  the data using nominatim replication service , whether indexing will done automatically or we need to initialize the indexing manually after every new update

**Comment by Stefanic** [comment]
No, indexing is part of the update process.
